scripts/import-item-cleanup/config.ini

```diff
@@ -0,0 +1,7 @@
+[args]
```

Comment:
a bit strange that this is in config dig v. openlibrary/config or...

Why don't we take in a config file as a param for the script and then show this config as a doc in the file above, e.g.

Here is what a valid config looks like:
```
in_file=./import-item-cleanup/in.txt
state_file=./import-item-cleanup/curline.txt
error_file=./import-item-cleanup/errors.txt
batch_size=1000
dry_run=True
```